index.html

```diff
@@ -12,52 +12,64 @@
             </div>
             <div v-else>
                 <h1>Welcome to Circle Game {{ name }}.</h1>
-            </div>
 
-            <div v-show = "name.length !== 0" class="game-box">
-                <div class="play-area">
-                    <label>Select BackGround Color of the Circle</label>
-                    <select v-model="circleColor">
-                        <option value="greenyellow" selected>Light Green</option>
-                        <option value="red">Red</option>
-                        <option value="blue">Blue</option>
-                        <option value="purple">Purple</option> 
-                        <option value="blue">Blue</option>
-                        <option value="yellow">yellow</option>
-                    </select>
+                <div class="game-box">
+                    <div class="play-area">
+                        <label>Select BackGround Color of the Circle</label>
+                        <select v-model="circleColor">
+                            <option value="greenyellow" selected>Light Green</option>
+                            <option value="red">Red</option>
```

Comment:
Here we can defined color list in 1 variables and then loop through to avoid duplicate code

Reply:
Got it Vicky bhai. I was also thinking to get some solution for it.